swagger.yaml

```diff
@@ -721,13 +727,78 @@ paths:
                 items:
                   $ref: "#/components/schemas/Error"
 
+  "/v1/users/{userGuid}/flextime": 
```

Comment:
This looks like the severa endpoint. But we don't need to define that in this spec. Rather I would think this could be something like severa/flexitime/{severaGuid}. 

We should also consider at this point is it possible/ necessary to have an option to not pass an id and get all users flexitime. This could be a nice admin feature but depends on the severa API. If the severa flexitime endpoint always requires an id and does not return a list then we can not worry about this for now and just implement it with the endpoint as I just described above.

Reply:
I think this Severa endpoint just returns flextime from one user. It needs to have GUID and optional date. 
I change userGuid to our find-user spec severaGuid.